src/main/java/org/jenkinsci/plugins/durabletask/PowershellScript.java

```diff
@@ -63,6 +64,11 @@ public void setPowershellBinary(String powershellBinary) {
         this.powershellBinary = powershellBinary;
     }
 
+    public boolean isLoadProfile() { return loadProfile; }
```

Comment:
use line breaks, to match the code style in the file
```suggestion
    public boolean isLoadProfile() {
        return loadProfile; 
    }
```